hapi/hedera-protobufs/services/state/roster/roster.proto

```diff
@@ -103,5 +88,5 @@ message RosterEntry {
      * SHALL NOT supply both values for the same endpoint.<br/>
      * This list SHALL NOT be empty.<br/>
      */
-    repeated proto.ServiceEndpoint gossip_endpoint = 5;
+    repeated proto.ServiceEndpoint gossip_endpoint = 4;
```

Comment:
Renumbering a field is the most incompatible change you can make to a protobuf definition, and cannot be "fixed" with migration.
Are we certain there is no data anywhere that would require migration for this change?

Reply:
@jsync-swirlds, All writes of the roster data structure to state are gated behind feature flags that have not been turned on by default yet.

Reply:
And they are not turned on in testnet, or previewnet?
Just making sure we don't create a situation where we parse data in a way that _silently_ corrupts data somewhere.

Reply:
Reverting to reserving the field as per our discussion, thanks @edward-swirldslabs !